server/v1/controllers/shows/instance/remove.js

```diff
@@ -1,17 +1,27 @@
-const db = require('../../../models');
+const {
+  utils: { getModelForEventType },
+} = require('../utils');
 const { master_time_id__byShowType } = require('../utils/utils__mongoose');
 
 async function remove(req, res) {
+  const { eventType } = req.params;
+
+  const dbModel = getModelForEventType(eventType);
+  if (!dbModel) {
+    res.send(404);
+    return;
+  }
+
```

Comment:
This code block (lines 7-14) have been added to all of the controllers for shows. `dbModel` is set to either `db.Show` or `db.Traffic` depending on the route.